settings.py

```diff
@@ -28,6 +28,10 @@
 SITE_ID = 1
 USE_TZ = True
 
+from django.utils.crypto import get_random_string
+chars = 'abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)'
+SECRET_KEY = get_random_string(50, chars)
```

Comment:
This should be in config files also we need to figure out the proper way to generate this for edx.